Tests/SonarQube.MSBuild.Tasks.UnitTests/MergeRuleSetsTests.cs

```diff
@@ -1,29 +1,20 @@
 ﻿//-----------------------------------------------------------------------
-// <copyright file="MergeRulesetTests.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="MergeRuleSetsTests.cs" company="SonarSource SA and Microsoft Corporation">
 //   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
 //   Licensed under the MIT License. See License.txt in the project root for license information.
 // </copyright>
 //-----------------------------------------------------------------------
 
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Xml.Linq;
 using TestUtilities;
 
 namespace SonarQube.MSBuild.Tasks.UnitTests
 {
     [TestClass]
-    public class MergeRulesetTests
+    public class MergeRuleSetsTests
```

Comment:
These tests haven't changed: I just moved the ruleset-checking assertion methods into a new shared class.